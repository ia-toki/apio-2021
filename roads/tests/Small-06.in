c10234cabdfd6eae47773fb6f163e0350fc797e1
200
15 164 908760209
0 78 964622612
167 110 977573022
123 165 965418004
53 164 959960110
198 185 915250829
140 168 976448278
41 133 984136170
108 103 994961811
124 123 995127420
116 159 951367854
192 83 981333268
104 159 934640323
103 34 906191523
26 159 922521726
30 78 963354979
199 133 970538960
101 22 944495693
125 158 924324450
128 159 913348366
12 164 956112278
136 181 937516519
59 172 929177237
138 31 916914592
65 47 929984053
135 114 966699198
143 49 927551518
49 77 934863321
105 47 901715915
92 125 952978300
73 116 967335805
148 114 925644996
17 164 902426787
54 114 956324377
109 171 983698280
10 186 946972791
176 159 955304431
86 47 900088283
37 47 985588883
4 164 960702181
74 117 980397867
63 154 993229923
147 133 979475510
93 133 962653025
18 88 917312384
79 9 941345353
32 133 984529808
25 114 977028657
58 186 955746912
87 60 959356830
42 51 981150345
154 78 985893745
8 107 989633862
172 78 958647253
45 17 918318199
166 47 970950198
130 133 922860099
186 34 970281244
69 76 913040277
90 133 978488358
44 146 959634884
78 34 931991983
95 102 902356883
3 78 942783563
21 78 941360504
146 47 954015255
161 159 911511694
150 21 916673870
117 159 915672405
96 111 911632857
35 53 967923913
142 190 954473226
22 78 946239734
80 78 964219829
46 159 937632909
164 34 934129462
193 144 901877775
173 34 985062126
81 39 948801862
83 104 925294453
127 179 915931770
119 9 981008664
24 186 917516746
152 59 904502539
9 47 946066443
52 108 966246125
179 168 966631563
114 34 925470576
133 34 991443982
6 159 959558614
197 88 971276799
168 47 941895493
185 159 945986016
144 114 948455225
38 78 959725942
110 133 974023177
190 78 967593536
11 64 965555772
7 114 920505608
182 114 943483125
51 114 919463225
40 133 984741085
196 105 984587081
177 83 960845131
75 66 962897944
13 114 924924764
102 114 967450659
169 3 982369144
5 133 966021078
47 34 928127576
84 166 990149674
156 114 945807539
29 116 966255904
122 47 946640016
19 51 945386974
155 164 954316624
107 17 950871307
121 196 968508850
33 35 964727878
183 164 961480399
36 114 936049854
112 157 952653246
134 175 933000853
60 57 932497792
132 191 911236130
194 13 956434081
34 171 998340972
178 104 917211436
76 159 999729766
28 164 951534666
165 186 966226100
39 161 902980694
27 136 948625329
111 47 945424626
56 123 935546033
158 164 974969616
57 47 950902325
2 28 911725292
189 35 901599670
50 72 986961341
43 108 935772459
151 47 953451334
188 93 958205175
14 83 938954839
67 133 997130053
170 186 926743894
71 34 985451320
72 164 908351925
184 159 926117736
62 117 933330906
149 154 911543606
131 3 986635861
99 182 925878394
180 183 917121521
48 186 939911527
98 78 909076794
61 79 958206829
157 78 994555875
160 133 994152659
97 193 956657022
68 70 971493698
66 166 964313708
174 0 973360394
145 47 982718390
175 114 974693062
70 164 966521625
195 160 944565578
115 103 916305854
187 186 982966408
85 34 917125449
106 185 944128457
181 165 905753242
163 95 922467255
91 175 982900010
89 47 949301235
129 135 939604845
16 78 951439002
1 34 914666235
23 162 931090822
31 34 971300918
82 164 900347268
88 186 900619113
153 175 934390083
191 159 982449391
77 95 957721422
113 159 998981485
55 88 959892071
162 85 981157975
100 127 944725334
139 22 992326351
118 37 911723499
159 34 953176240
137 114 923394546
141 31 999587560
94 165 925895313
126 44 928793412
64 133 938538641
120 69 949828290
20 164 901005798
