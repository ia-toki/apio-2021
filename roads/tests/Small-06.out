064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
188936224485 127454961384 89577027303 75298363278 66637420275 59006987048 51415908748 43858574765 37176332375 30602849218 24957582335 19322064872 13737648201 8203882220 2731063202 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
